backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/DefaultAndroidInput.java

```diff
@@ -221,15 +223,24 @@ public float getGyroscopeZ () {
 	}
 
 	@Override
-	public void getTextInput (final TextInputListener listener, final String title, final String text, final String hint) {
+	public void getTextInput (TextInputListener listener, String title, String text, String hint) {
+		getTextInput(listener, title, text, hint, OnscreenKeyboardType.Default);
+	}
+
+	@Override
+	public void getTextInput(final TextInputListener listener, final String title, final String text, final String hint, final OnscreenKeyboardType type) {
 		handle.post(new Runnable() {
 			public void run () {
 				AlertDialog.Builder alert = new AlertDialog.Builder(context);
 				alert.setTitle(title);
 				final EditText input = new EditText(context);
+				input.setInputType(getAndroidInputType(type));
```

Comment:
Since this uses EditText, would specifying `InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_FLAG_AUTO_CORRECT | InputType.TYPE_TEXT_FLAG_CAP_SENTENCES` to enable auto suggest and auto correct work?

Reply:
Thanks for the hint. Auto correction and suggestions were already present before, but the change removed it. I changed the behaviour to only overwrite the input type if something else than the default keyboard is needed.